tests/110-test-fail-if-no-thinpool.sh

```diff
@@ -0,0 +1,36 @@
+source $SRCDIR/libtest.sh
+
+test_fail-if-no-thinpool() {
+  local devs dev
+  local devlinks devlink
+  local test_status=1
+  local testname=`basename "$0"`
+  local vg_name="dss-test-foo"
+  local infile=${WORKDIR}/container-storage-setup
+  local outfile=${WORKDIR}/container-storage
+
+  # Error out if any pre-existing volume group vg named dss-test-foo
+  if vg_exists "$vg_name"; then
+    echo "ERROR: $testname: Volume group $vg_name already exists." >> $LOGS
+    return $test_status
+  fi
+
+  cat << EOF > $infile
+DEVS="$devs"
+VG=$vg_name
+EOF
+  # Run docker-storage-setup
+  $DSSBIN $infile $outfile >> $LOGS 2>&1
+
+  # Test failed.
+  if [ $? -ne 0 ]; then
+      test_status=0
+  else
+      echo "ERROR: $testname: $DSSBIN Succeeded. Should have failed with CONTAINER_THINPOOL specified" >> $LOGS
+  fi
+  cleanup $vg_name "$DEVS" "$infile" "$outfile"
+  return $test_status
+}
+
+# Make sure command fails if no CONTAINER_THINPOOL is specified
+test_fail-if-no-thinpool
```

Comment:
Typically all function names have underscore as separator. This seems to have both underscore and hyphen as separators. Would be good to have consistent usage of underscore.